src/components/ChallengeModal/ChallengeModal.tsx

```diff
@@ -769,6 +789,40 @@ export class ChallengeModal extends Modal<Events, ChallengeModalProperties, any>
                         </div>
                     </div>
                 )}
+                {(mode === "open" || null) && (
+                    <>
+                        <div
+                            className={
+                                "form-group" + (this.state.challenge.game.rengo ? "" : " hide")
+                            }
+                        >
+                            <label className="control-label" htmlFor="rengo-casual-mode">
+                                {_("Casual")}
+                            </label>
+                            <div className="controls">
+                                <div className="checkbox">
+                                    <input
+                                        type="checkbox"
+                                        id="rengo-casual-mode"
+                                        checked={this.state.challenge.game.rengo_casual_mode}
+                                        onChange={this.update_rengo_casual}
+                                    />
+                                </div>
+                            </div>
+                        </div>
+                        <div
+                            className={
+                                "casual-time-warning" +
+                                (this.state.challenge.game.rengo &&
+                                this.state.challenge.game.rengo_casual_mode
+                                    ? ""
+                                    : " hide")
+                            }
+                        >
+                            *{_("use Simple time")}
```

Comment:
We need to enforce this by removing the option, no one reads :sweat_smile:

Reply:
Note: the server rejects casual mode challenges with non-simple time, so if they don't read, they get a user friendly worded error dialog to this effect when they hit "Create Challenge".

(Not saying it couldn't be better, just that its not dire 😁  )

Reply:
Done.